compilers/concrete-compiler/compiler/include/concretelang/ClientLib/ClientLib.h

```diff
@@ -33,6 +34,13 @@ using concretelang::values::Value;
 namespace concretelang {
 namespace clientlib {
 
+Result<TransportValue>
+importTfhersFheUint8(llvm::ArrayRef<uint8_t> serializedFheUint8);
+Result<std::vector<uint8_t>> exportTfhersFheUint8(TransportValue value,
+                                                  TfhersFheIntDescription info);
+Result<TfhersFheIntDescription>
+getTfhersFheUint8Description(llvm::ArrayRef<uint8_t> serializedFheUint8);
+
```

Comment:
I think this would be better put into `Commons`. If ever we need to use that in the `ServerLib` as well.